MANIFEST.in

```diff
@@ -1,15 +1,19 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
```

Comment:
I am wondering, whether this information will be helpful, as it will be not accurate anymore with new commits. Maybe we should include the commit-id already here?

Reply:
I find it helpful.

I don't know about the exact commit ID, it'll be in the .meta.cfg, do we want it repeated everywhere?